drivers/include/at86rf2xx.h

```diff
@@ -89,14 +90,15 @@ extern "C" {
  * @brief   Flags for device internal states (see datasheet)
  * @{
  */
-#define AT86RF2XX_STATE_TRX_OFF      (0x08)     /**< idle */
-#define AT86RF2XX_STATE_PLL_ON       (0x09)     /**< ready to transmit */
-#define AT86RF2XX_STATE_SLEEP        (0x0f)     /**< sleep mode */
-#define AT86RF2XX_STATE_BUSY_RX_AACK (0x11)     /**< busy receiving data */
-#define AT86RF2XX_STATE_BUSY_TX_ARET (0x12)     /**< busy transmitting data */
-#define AT86RF2XX_STATE_RX_AACK_ON   (0x16)     /**< wait for incoming data */
-#define AT86RF2XX_STATE_TX_ARET_ON   (0x19)     /**< ready for sending data */
-#define AT86RF2XX_STATE_IN_PROGRESS  (0x1f)     /**< ongoing state conversion */
+#define AT86RF2XX_STATE_FORCE_TRX_OFF  (0x03)     /**< force transition to idle */
```

Comment:
@DipSwitch I think you have a point here, still I am sceptic to use you proposed approach. The `FORCE_TRX_OFF` command is and shout only be used internally by the drive. How about the following, (i) remove the `AT86RF2XX_STATE_FORCE_TRX_OFF` from the `at86rf2xx.h` header and use the command defined in `at86rf2xx_registers.h` and (ii) properly comment why we use it in `at86rf2xx_reset_state_machine`.